CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#563](https://github.com/rubocop/rubocop-rails/issues/563): Improve wording for `Rails/UniqueValidation` description. ([@andyw8][])
```

Comment:
Can you remove this changelog entry? This is not a change of the cop behavior.

Reply:
Sure, done. (but I'd argue this _is_ a change to the user-observable behaviour, although very minor).

Reply:
> although very minor

That's exactly right.